ddtrace/llmobs/_integrations/gemini.py

```diff
@@ -39,13 +39,14 @@ def _llmobs_set_tags(
         kwargs: Dict[str, Any],
         response: Optional[Any] = None,
         operation: str = "",
+        history: List[Any] = [],
     ) -> None:
         span.set_tag_str(SPAN_KIND, "llm")
         span.set_tag_str(MODEL_NAME, span.get_tag("google_generativeai.request.model") or "")
         span.set_tag_str(MODEL_PROVIDER, span.get_tag("google_generativeai.request.provider") or "")
 
         instance = kwargs.get("instance", None)
-        metadata = self._llmobs_set_metadata(kwargs, instance)
+        metadata = llmobs_get_metadata_google(kwargs, instance)
         span.set_tag_str(METADATA, safe_json(metadata))
 
         system_instruction = _get_attr(instance, "_system_instruction", None)
```

Comment:
I have updated the helper method... it should now work for both the gemini and vertex ai cases. Since I need to check for the `google.ai.generativelanguage_v1beta.types.content.Content` type, I had to add `google-ai-generativelanguage` as a dependency in riotfile,py for both gemini and vertexai